javascript/asynchronous_javascript_and_apis/working_with_apis.md

```diff
@@ -15,42 +15,52 @@ This section contains a general overview of topics that you will learn in this l
 
 Servers that are created for serving data for external use (in websites or apps) are often referred to as APIs or ['Application Programming Interfaces'](https://www.youtube.com/watch?v=s7wmiS2mSXY).
 
-There are multiple ways of requesting data from an API, but all of them basically do the same thing. For the most part, APIs are accessed through URLs, and the specifics of how to query these URLs change based on the specific service you are using. For example, WeatherAPI has several types of data that you can request. To get the current weather in a specific location, you can pass in the name of a city (optionally, you can also pass a zip code & even an ip-address!) as a URL query string parameter, like so:
+There are multiple ways of requesting data from an API, but all of them basically do the same thing. For the most part, APIs are accessed through URLs, and the specifics of how to query these URLs change based on the specific service you are using. For example, Visual Crossing has several types of data that you can request. To get the current weather in a specific location, you can pass in the name of a city as a URL query string parameter, like so:
 
 ```text
-https://api.weatherapi.com/v1/current.json?q=london
+https://weather.visualcrossing.com/VisualCrossingWebServices/rest/services/timeline/london
 ```
 
-The specifics for using any API are usually documented on the service's website. Check the [WeatherAPI documentation](https://www.weatherapi.com/docs/). If you haven't already, go ahead and paste the weather URL above, with the city of your choice, into your browser... (we'll wait).
+The specifics for using any API are usually documented on the service's website. Check the [Visual Crossing API Documentation](https://www.visualcrossing.com/resources/documentation/weather-api/timeline-weather-api/). If you haven't already, go ahead and paste the weather URL above, with the city of your choice, into your browser... (we'll wait).
```

Comment:
```suggestion
The specifics for using any API are usually documented on the service's website. Check out the [Visual Crossing API Documentation](https://www.visualcrossing.com/resources/documentation/weather-api/timeline-weather-api/). If you haven't already, go ahead and paste the weather URL above, with the city of your choice, into your browser... (we'll wait).
```

Reply:
I read and understood all your suggestions, thank you for your time in reviewing this! So I can directly commit the suggestions from here, yes?